python/RappCloud/CloudMsgs/ObjectDetectionClearModels.py

```diff
@@ -0,0 +1,72 @@
+from RappCloud.Objects import (
+    File,
+    Payload)
+
+from Cloud import (
+    CloudMsg,
+    CloudRequest,
+    CloudResponse)
+
+
+class ObjectDetectionClearModels(CloudMsg):
+    """ Object Detection - Clear Models CloudMsg object"""
+
+    class Request(CloudRequest):
+        """ Object Detection - Clear Models Cloud Request object. ObjectDetectionClearModels.Request """
+        def __init__(self, **kwargs):
+            """!
+            Constructor
+
+            @param **kwargs - Keyword arguments. Apply values to the request attributes.
+                - @ref user
+            """
+
+            ## Username 
+            self.user = 'rapp'
```

Comment:
To remove. User information is retrieved on the RAPP Platform using the user's credentials (token).
Resource [here](https://github.com/rapp-project/rapp-platform/wiki/How-to-create-a-HOP-service-for-a-ROS-service%3F#web-service-implementation)